tests/entrypoints/test_openai_server.py

```diff
@@ -694,6 +694,7 @@ async def test_guided_grammar(server, client: openai.AsyncOpenAI):
         prompt=("Generate a sql state that select col_1 from "
                 "table_1 where it is equals to 1"),
         temperature=1.0,
+        n=3,
```

Comment:
since n = 3 here let's also verify all the outputs, you might also need to add `use_beam_search` to `extra_body`